governance/roles.md

```diff
@@ -96,9 +96,42 @@ TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/maste
 
 ## Facilitation roles
 
-Facilitation roles are identified in [github settings](/.github/settings.yaml)
-and include additional permissions in the repo which are governed by policies
-described below.
+Members often contribute by working in small groups that research and discuss
+options and then share their findings with the rest of the group in a
+presentation and typically contribute to the body of work in this
+repository (via Pull Request). Some tasks can be effectively executed
+by simply chiming in on a github issue and independently contributing a PR
+without any particular role or authorization.
+
+This section describes roles where more coordination is helpful and the
+work benefits from identifying one or more Members who will take
+on specific responsibilities.  Our goal is to empower Members to move forward
+independently, while defining an approval process designed to support
+communication and alignment across the wider group.
+
+New roles in this section are typically defined after someone has acted in
+that role for some time informally, then proposed as an on-going role by Pull
+Request which must be approved by a majority of Chairs.  Process clarifications
+are welcome and PRs can be approved by any single Chair.
+
+Members who take on Roles are responsible for
+understanding the approval policies related to their role and ensure that
+commits to the repo are reviewed in accordance with our [governance](./)
```

Comment:
not sure if "governance" is the same as the approval policies in line 118, if so we should be more clear "...the approval policies, as defined in our [governance](./),

Reply:
well, the security reviewer role is defined outside of /governance -- just trying to say that anyone with a role that involves commits to the repo is responsible for anything that is written in our governance (e.g. consistent with charter and mission, transparency, inclusion, and anything else that we wrote down as overall policy for how we work together as a group)

Feel free to suggest alternate wording, but I don't want to imply less responsibility for people who take on roles.  I want people to feel an obligation to fully understand and commit to taking ownership of understanding our overall governance and acting appropriately.

Reply:
maybe I'll pull this out as an issue, so we can get the rest of the PR improvements committed...  if other folks agree this PR is net-positive

Reply:
see https://github.com/cncf/sig-security/issues/233